addon/components/nypr-account-forms/login.js

```diff
@@ -47,7 +53,29 @@ export default Component.extend({
     },
     loginWithFacebook() {
       get(this, 'session').authenticate('authenticator:torii', 'facebook-connect')
-      .catch(() => {});
+      .then(() => {
+        if (this.get('session').get('isNewSocialUser') === true) {
```

Comment:
right, now we can just catch on the failure case and everything else should be handled by ember simple auth